gtda/homology/simplicial.py

```diff
@@ -110,20 +111,20 @@ class VietorisRipsPersistence(BaseEstimator, TransformerMixin, PlotterMixin):
     <https://github.com/scikit-tda/ripser.py>`_ package.
 
     `GUDHI <https://github.com/GUDHI/gudhi-devel>`_ is used as a C++ backend
-    for the edge collapse algorithm described in [2]_.
+    for the edge collapse algorithm described in Boissonnat and Pitram [2]_ .
 
     References
     ----------
-    [1] U. Bauer, "Ripser: efficient computation of Vietoris–Rips persistence \
-        barcodes", 2019; `arXiv:1908.02518 \
-        <https://arxiv.org/abs/1908.02518>`_.
+    .. [1] U. Bauer, "Ripser: efficient computation of Vietoris–Rips \
```

Comment:
The missing dots were deliberate. The issue was that there is no reference to this bib entry in the text (no [1]_). But I now think there should be, in the first of the Notes above:

`Ripser <https://github.com/Ripser/ripser>`_ [1]_ is used as a C++ backend for ...

or something similar